Buildings/Experimental/DHC/EnergyTransferStations/Combined/Generation5/Controls/SideCold.mo

```diff
@@ -0,0 +1,254 @@
+    annotation (Placement(transformation(extent={{-220,20},{-180,60}}),iconTransformation(extent={{-140,
+            -62},{-100,-22}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput TBot(
+    final unit="K",
+    displayUnit="degC")
+    "Temperature at bottom of tank"
+    annotation (Placement(transformation(extent={{-220,-100},{-180,-60}}),
+                                                                         iconTransformation(extent={{-140,
+            -104},{-100,-64}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput yAmb[nSouAmb](
+    each final unit="1")
+    "Control signal for ambient sources"
+    annotation (Placement(transformation(extent={{180,20},{220,60}}),iconTransformation(extent={{100,20},{140,60}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput yValIso(
+    final unit="1")
+    "Ambient loop isolation valve control signal"
+    annotation (Placement(transformation(extent={{180,-20},{220,20}}),iconTransformation(extent={{100,-20},{140,20}})));
+  Buildings.Controls.OBC.CDL.Conversions.BooleanToReal booToRea
+    "Convert DO to AO signal"
+    annotation (Placement(transformation(extent={{40,-110},{60,-90}})));
+  Buildings.Controls.OBC.CDL.Continuous.GreaterThreshold greThr(t=0.01)
+    "Control signal is non zero (with 1% tolerance)"
+    annotation (Placement(transformation(extent={{0,-110},{20,-90}})));
+  Modelica.Blocks.Discrete.ZeroOrderHold zeroOrderHold(samplePeriod=60)
+    annotation (Placement(transformation(extent={{80,-110},{100,-90}})));
+equation
+  connect(x1.y,mapFun.x1)
+    annotation (Line(points={{22,20},{36,20},{36,8},{98,8}},
+                                                         color={0,0,127}));
+  connect(rep.y,mapFun.u)
+    annotation (Line(points={{-8,0},{98,0}},                   color={0,0,127}));
+  connect(f1.y,mapFun.f1)
+    annotation (Line(points={{22,-20},{36,-20},{36,4},{98,4}},color={0,0,127}));
+  connect(f2.y,mapFun.f2)
+    annotation (Line(points={{12,120},{40,120},{40,-8},{98,-8}},color={0,0,127}));
+  connect(x2.y,mapFun.x2)
+    annotation (Line(points={{22,-60},{44,-60},{44,-4},{98,-4}},
+                                                               color={0,0,127}));
+  connect(TSet,conTChiWatSup.u_s)
+    annotation (Line(points={{-200,40},{-148,40},{-148,-20},{-130,-20}},color={0,0,127}));
+  connect(TBot,conTChiWatSup.u_m)
+    annotation (Line(points={{-200,-80},{-118,-80},{-118,-32}},                      color={0,0,127}));
+  connect(f2[1].y,mapFunTChiSupSet.x2)
+    annotation (Line(points={{12,120},{40,120},{40,76},{98,76}},color={0,0,127}));
+  connect(minTChiWatSup.y,mapFunTChiSupSet.f2)
+    annotation (Line(points={{84,60},{90,60},{90,72},{98,72}},color={0,0,127}));
+  connect(TSet,mapFunTChiSupSet.f1)
+    annotation (Line(points={{-200,40},{20,40},{20,84},{98,84}},color={0,0,127}));
+  connect(conTChiWatSup.y,addPar.u)
+    annotation (Line(points={{-106,-20},{-106,-18},{-104,-18},{-104,-20},{-100,
```

Comment:
eliminate the unnecessary points `{-106,-18},{-104,-18},{-104,-20}`

Reply:
Addressed